src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,37 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static final String regex = "[,:]";
+    private static final String custom_delimiter = "//(.)\n(.*)";
```

Comment:
```suggestion
    private static final String REGEX = "[,:]";
    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
```
상수 활용 👍 

자바에서의 상수는 대문자가 컨벤션입니다.